recipes/cd3-boost-unit-definitions/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import (
+    apply_conandata_patches,
+    export_conandata_patches,
+    get,
+    copy,
+)
+from conan.tools.layout import cmake_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "cd3-boost-unit-definitions"
+    description = "A collection of pre-defined types and unit instances for working with Boost.Units quantities."
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/CD3/BoostUnitDefinitions"
+    # no "conan" and project name in topics. Use topics from the upstream listed on GH
+    # Keep 'hearder-only' as topic
+    topics = ("physical dimensions", "header-only")
+    settings = "os", "arch", "compiler", "build_type"  # even for header only
+    no_copy_source = True  # do not copy sources to build folder for header only projects, unless, need to apply patches
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    # in case the project requires C++14/17/20/... the minimum compiler version should be listed
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "14.1",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "5.1",
+        }
+
+    # no exports_sources attribute, but export_sources(self) method instead
+    # this allows finer grain exportation of patches per version
+    def export_sources(self):
+        # export_conandata_patches(self)
+        pass
+
+    def layout(self):
+        # src_folder must use the same source folder name the project
+        cmake_layout(self)
```

Comment:
As this is header only and not using the CMake helpers we can use the `basic_layout`

```suggestion
        basic_layout(self)
```